src/data/content/design-system/componenti/card.yaml

```diff
@@ -83,7 +83,22 @@ tabs:
             title: Come usarlo
             headingLevel: 2
             text: |
-              _In stesura_
+              
+              **Rendi le card azionabili.** Poiché sono una sintesi di informazioni più dettagliate, ogni singola card deve puntare a queste informazioni con un link.
+
+              **Includi contenuti unici.** Evita di ripetere immagini o informazioni che sono presenti in molte card. Questo aiuta a rendere ogni carta unica.
+
+              ### Attenzione a
```

Comment:
Aggiungerei due punti di attenzione: (1) l'uso della corretta semantica heading, se possibile non cambiando peso ai titoli, per favorire la leggebilità ai lettori di schermo e l'usabilità in generale e (2) se raggruppate molte card (es. listato) il valutare di racchiuderle in elementi lista per permetterne lo scorrimento con cognizione di causa.